.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -148,13 +148,11 @@ body:
         **You are not obliged to answer this question**.
         We do encourage answering though as it provides context to better assist you.
         Less experienced users tend to make common mistakes, which is ok; by letting us know we can spot those more easily.
-        
+
       options:
         - label: I am inexperienced with docker
         - label: I am inexperienced with mail servers
         - label: I am uncomfortable with the CLI
-    validations:
-      required: false
   - type: checkboxes
```

Comment:
Looks like this was all migrated to the other PR and doesn't belong here? (_I'm not sure if the line range is showing up for this comment, but all changes in this file_)

Reply:
- There are more leftovers than I thought, this s also one...

Reply:
Done in 3c7483149d53e750925fe5651df3fadac1ae239e